src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -5566,6 +5568,247 @@ void GetVRFInputData(EnergyPlusData &state, bool &ErrorsFound)
     }
 }
 
+void CheckVRFTUNodeConnections(EnergyPlusData &state, int const VRFTUNum, bool &ErrorsFound)
```

Comment:
Would it make sense to make this a member function and remove `VRFTUNum` as an argument?
And make this a `bool` function and remove `ErrorsFound` as an argument?

Reply:
This function is inside GetVRFInputData which is not a member function. I think when models gather all instances of the object in a for loop, instead of just getting the current object data, then non-member functions will be used.

Re a bool function:

ErrorsFound is set throughout GetVRFInputData and then at the end this new function is called to test the node connections (within the for loop for all TUs). If this were a bool function then the call would look like this:

ErrorsFound = ErrorsFound || CheckVRFTUNodeConnections(EnergyPlusData &state, int const VRFTUNum);

I guess either way would work. Not sure I like the implications of this syntax where developers may forget to include "ErrorsFound || boolFunctionCall" to retain the current value of ErrorsFound.

Reply:
That's fine as-is.